### ISSUE TITLE
Make hover tooltip show up automatically after specified delay

### ISSUE THREAD

**Issue by krassowski** [jupyter-lsp/jupyterlab-lsp#362]
At the moment the user has to press <kbd>Ctrl</kbd> (or a different modifier key if changed in settings) to bring up the hover tooltip. We could allow specifying a delay as an alternative/additional activation method. This is how it is implemented in VScode I think.

Sketch for setting schema:

```json
"delay": {
  "title": "Hover delay",
  "type": ["number", "null"],
  "default": 300,
  "description": "Number of milliseconds after which the hover tooltip should be shown. Set to `null` to disable the automatic display of the tooltip (it will still be possible to bring it up with the modifier key)."
},
```

The following files are relevant:
- [packages/jupyterlab-lsp/src/features/hover.ts](https://github.com/krassowski/jupyterlab-lsp/blob/master/packages/jupyterlab-lsp/src/features/hover.ts) - for the implementation
- [packages/jupyterlab-lsp/schema/hover.json](https://github.com/krassowski/jupyterlab-lsp/blob/master/packages/jupyterlab-lsp/schema/hover.json) - for schema updates
- [atest/05_Features/Hover.robot](https://github.com/krassowski/jupyterlab-lsp/blob/master/atest/05_Features/Hover.robot) - for tests

Questions to the users:
- should it be enabled by default?
- what should be the default delay?

**Comment by yamaton** [comment]
Let's re-ignite this auto-tooltip discussion.

>  * should it be enabled by default?

Yes. A consistent UI/UX is what I like.

> * what should be the default delay?

300 ms, the default value in VS code.

**Comment by krassowski** [comment]
Closing as this was implemented in #864. Thank you again!
